content/webapp/pages/exhibition-guide.tsx

```diff
@@ -309,27 +317,26 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
                 {type === 'audio-with-descriptions' &&
                   audioWithDescription?.url && (
                     <AudioPlayer
-                      title={
-                        stop.title
-                          ? `${number}. ${stop.title}`
-                          : `${number}. ${standaloneTitle}`
-                      }
+                      title={getTitle(stop)}
                       audioFile={audioWithDescription.url}
                     />
                   )}
                 {type === 'audio-without-descriptions' &&
                   audioWithoutDescription?.url && (
                     <AudioPlayer
-                      title={
-                        stop.title
-                          ? `${number}. ${stop.title}`
-                          : `${number}. ${standaloneTitle}`
-                      }
+                      title={getTitle(stop)}
                       audioFile={audioWithoutDescription.url}
                     />
                   )}
                 {type === 'bsl' && bsl.embedUrl && (
-                  <VideoEmbed embedUrl={bsl.embedUrl} />
+                  <figure className="no-margin">
```

Comment:
I think there was maybe a move to remove utility classes like `no-margin` – not sure where we are with it or if we decided `no-margin` should continue to exist cc. @rcantin-w ?

Reply:
The difficulty with `no-margin` is that it's styled as `margin: 0!important;`, so when refactoring, it's just about checking that the `!important` isn't actually required - which might mean changes to another component. But yeah if we can remove it, it's always a win!